### ISSUE TITLE
XML Parsing Error: not well-formed

### ISSUE THREAD

**Issue by longtimeago** [Nerodesk/nerodesk#224]
I tried to upload a file on dev environment (mvn clean integration-test -Phit-refresh -Dport=8080)
and got html page with following content

```
XML Parsing Error: not well-formed
Location: http://localhost:8080/
Line Number 1, Column 5:

HTTP/1.1 500 Internal Error
----^
```

Logs contain only one new line

```
com.nerodesk.om.mock.MkDoc: /tmp/1428741582982-0/urn/test/1/starter.zip saved
```

**Comment by longtimeago** [comment]
@yegor256 have you seen this? It's still actual

**Comment by yegor256** [comment]
@longtimeago yeah, it was, I fixed it in 6bd969b (it was a bug in takes)

**Comment by longtimeago** [comment]
@yegor256 thanks, I'll check it tonight
